src/utilities/tests/test_ck_utilities.py

```diff
@@ -60,10 +62,13 @@ def test_ck_read_write(show_plots):
     timeWrite = scObjectLogger.times()
     sigmaWrite = scObjectLogger.sigma_BN
     omegaWrite = scObjectLogger.omega_BN_B
-    pyswice_ck_utilities.ckWrite("test.bc", timeWrite, sigmaWrite, omegaWrite, timeInit, spacecraft_id=-202)
+
+    tempDirectory = tempfile.TemporaryDirectory()
```

Comment:
We should either call `.cleanup()` on this object at some point, or -- and more preferably -- use it in a `with` block to ensure proper cleanup automatically.

Reply:
Made this change. Thanks for the heads up.